pyproject.toml

```diff
@@ -64,6 +64,7 @@ sdgym = { main = 'sdgym.cli.__main__:main' }
 
 [project.optional-dependencies]
 dask = ['dask', 'distributed']
+realtabformer = ['realtabformer>=0.2.1']
```

Comment:
This has to be added to `test`:
```
test = [
    'sdgym[realtabformer]',
    ...
]
```

Reply:
`realtabformer = ['realtabformer>=0.2.1', 'transformers<=4.42']` because if the transformers are higher version it crashes as per https://github.com/worldbank/REaLTabFormer/issues/90